README.md

```diff
@@ -1,65 +1,49 @@
 
-Install with npm:
+![verdaccio gif](https://github.com/verdaccio/verdaccio/blob/master/assets/gif/verdaccio_big_30.gif?raw=true)
 
-```bash
-npm install --global verdaccio
-```
+# Version 4 
 
-Install with yarn:
+[Verdaccio](https://verdaccio.org/) is a simple, **zero-config-required local private npm registry**. 
+No need for an entire database just to get started! Verdaccio comes out of the box with 
+**its own tiny database**, and the ability to proxy other registries (eg. npmjs.org), 
+caching the downloaded modules along the way. 
+For those looking to extend their storage capabilities, Verdaccio 
+**supports various community-made plugins to hook into services such as Amazon's s3, 
+Google Cloud Storage** or create your own plugin.
 
-```bash
-yarn global add verdaccio
-```
 
-Install with pnpm:
+[![verdaccio (latest)](https://img.shields.io/npm/v/verdaccio/latest.svg)](https://www.npmjs.com/package/verdaccio)
+[![verdaccio (next)](https://img.shields.io/npm/v/verdaccio/next.svg)](https://www.npmjs.com/package/verdaccio)
+[![verdaccio (next)](http://img.shields.io/npm/dy/verdaccio.svg)](https://www.npmjs.com/package/verdaccio)
+[![docker pulls](https://img.shields.io/docker/pulls/verdaccio/verdaccio.svg?maxAge=43200)](https://verdaccio.org/docs/en/docker.html)
+[![backers](https://opencollective.com/verdaccio/tiers/backer/badge.svg?label=Backer&color=brightgreen)](https://opencollective.com/verdaccio)
+
+![circle ci status](https://circleci.com/gh/verdaccio/verdaccio.svg?style=shield&circle-token=:circle-token)
+[![codecov](https://img.shields.io/codecov/c/github/verdaccio/verdaccio/master.svg)](https://codecov.io/gh/verdaccio/verdaccio)
+[![discord](https://img.shields.io/discord/388674437219745793.svg)](http://chat.verdaccio.org/)
+[![node](https://img.shields.io/node/v/verdaccio/latest.svg)](https://www.npmjs.com/package/verdaccio)
+![MIT](https://img.shields.io/github/license/mashape/apistatus.svg)
+[![Crowdin](https://d322cqt584bo4o.cloudfront.net/verdaccio/localized.svg)](https://crowdin.com/project/verdaccio)
 
-```bash
-pnpm i -g verdaccio
-```
 
+[![Twitter followers](https://img.shields.io/twitter/follow/verdaccio_npm.svg?style=social&label=Follow)](https://twitter.com/verdaccio_npm)
+[![Github](https://img.shields.io/github/stars/verdaccio/verdaccio.svg?style=social&label=Stars)](https://github.com/verdaccio/verdaccio/stargazers)
 
-<h2 align="center">Introduction</h2>
 
-Verdaccio is a simple, **zero-config-required local private npm registry**. No need for an entire database just to get started! Verdaccio comes out of the box with **its own tiny database**, and the ability to proxy other registries (eg. npmjs.org), caching the downloaded modules along the way. For those looking to extend their storage capabilities, Verdaccio **supports various community-made plugins to hook into services such as Amazon's s3 and Google Cloud Storage**.
+## Install
+
+Install with npm:
```

Comment:
and if I want to use yarn? 😆

Reply:
For documentation I prefer stick with the official tooling. `yarn` and `pnpm` are awesome but users need and extra step (install them) while `npm` is already there.